plugins/chruby/chruby.plugin.zsh

```diff
@@ -16,14 +16,15 @@ _source-from-omz-settings() {
 }
 
 _source-from-homebrew() {
-  (( $+commands[brew] )) || return 1
-
   local _brew_prefix
   # check default brew prefix
-  if [[ -h /usr/local/opt/chruby ]];then
```

Comment:
This will continue to work on older Intel-based macs, as this Homebrew prefix is covered by checking the `/usr/local/share/chruby/chruby.sh` path [below](https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/chruby/chruby.plugin.zsh#L49), and the following symlinks exist on those macs (I've double-checked it on mine):

```
/usr/local/share/chruby -> /usr/local/Cellar/chruby/0.3.9/share/chruby
/usr/local/opt/chruby -> /usr/local/Cellar/chruby/0.3.9
```